nixos/tests/jellyfin.nix

```diff
@@ -4,6 +4,27 @@ import ./make-test-python.nix ({ lib, pkgs, ... }:
     name = "jellyfin";
     meta.maintainers = with lib.maintainers; [ minijackson ];
 
+    /* TODO: fix test script:
+      testScriptWithTypes:81: error: Invalid index type "str" for "str"; expected
+      type "Union[SupportsIndex, slice]"
+              auth_token = auth_result["AccessToken"]
+                                       ^
+      testScriptWithTypes:88: error: Invalid index type "str" for "str"; expected
+      type "Union[SupportsIndex, slice]"
+          ...ession for session in sessions_result if session["DeviceId"] == "1337"
+                                                              ^
+      testScriptWithTypes:152: error: Invalid index type "str" for "str"; expected
+      type "Union[SupportsIndex, slice]"
+              if item_info["Name"] != "Big Buck Bunny":
+                           ^
+      testScriptWithTypes:156: error: Invalid index type "str" for "str"; expected
+      type "Union[SupportsIndex, slice]"
+              media_source_id = item_info["MediaSources"][0]["Id"]
+                                          ^
+      Found 4 errors in 1 file (checked 1 source file)
+    */
+    skipTypeCheck = true;
```

Comment:
If you want to fix the typechecking, here is a patch you can apply: https://gist.github.com/minijackson/66f7b1f2b6699005e56d5ef8e5247422

If you don't want to modify your PR, no worries, I can make one afterwards.

Reply:
Applied your patch to this PR. Thank you.